README.md

```diff
@@ -121,6 +121,13 @@ locally if you are planning to contribute to the project.
   * For processing image files, `tesseract` is required. See [here](https://tesseract-ocr.github.io/tessdoc/Installation.html) for installation instructions.
   * For processing PDF files, `tesseract` and `poppler` are required. The [pdf2image docs](https://pdf2image.readthedocs.io/en/latest/installation.html) have instructions on installing `poppler` across various platforms.
 
+Additionally, if you're planning to contribute to `unstructured`, we provide you a `pre-commit` configuration
```

Comment:
"we provide you a `pre-commit` configuration" -> "we provide you an optional `pre-commit`" ?

^^ This optional, so let's be explicit about that.

Folks can still rely on (may prefer to) run `make tidy`, or let their editor do most of the work.

Reply:
Sure, I'll also mention the `make tidy` option :)